templates/default.mustache

```diff
@@ -1,18 +1,18 @@
-{{#progress}}
+{{#options.progress}}
```

Comment:
supposedly, this one would work without `options` as well, right? Did you update this to show people the possibility of using this or just to make something else work?

Reply:
You're correct. I think it might make sense to try and make the progress and controls a bit less "magical" and a bit more like something you could write yourself (i.e. in a theme) without having to modify any of the cleaver core files.

For that reason I'd also suggest deprecating/removing the `progress` and `options` variables here (this change would be required to make that happen) at some point.